### PR TITLE
AntiWorldBorder

### PR DESCRIPTION
**Pull request by dG9hc3Q**
Basically made an option to prevent world border shenanigans, rather than being combined with antiNetherRoof option.
Also fixing some grammar mistakes and added myself to the readme :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,14 @@
...
-Because of this paperbin can even be run on top of modified versions of paper.
+This project uses JVM TI to modify Minecraft classes at runtime, Because of this PaperBin can even be run on top of modified versions of paper.
 
 ## Donate
 You can donate by sending bitcoin to `bc1q35vyw5rudnttchglj3rch9p2j9xxannumn3muv`.
 
 ## Usage
 
-Note: It is highly recommended to run PaperBin on Java 15 using ZGC and large pages for best performance.
+Note: It is highly recommended running PaperBin on Java 15 using ZGC and large pages for best performance.
```
Comment (x4e) [inline README.md]: What is the purpose of this change?

File: README.md [hunk 2]
```diff
@@ -5,15 +5,14 @@
 
 An experiment at improving the performance of PaperMC. 
 
-This project uses jvmti to modify Minecraft classes at runtime.
-Because of this paperbin can even be run on top of modified versions of paper.
+This project uses JVM TI to modify Minecraft classes at runtime, Because of this PaperBin can even be run on top of modified versions of paper.
```
Comment (x4e) [inline README.md]: I'd prefer it if you kept the old wording.

File: README.md [hunk 3]
```diff
@@ -43,9 +42,17 @@ If it does not then quit it yourself once it has finished starting up the server
...
 
-## Test Server
-This plugin is currently active on `oldfag.org`, feel free to connect and test it out, there are no rules.
+(RemainingToast#3835)
+
+## Test Servers
+This plugin is currently active on 
+
+- `oldfag.org`
+- `2b2t.com.au`
```
Comment (x4e) [inline README.md]: Sorry, I'd prefer it if you did not add this server. Before adding any servers here I would want to properly review them myself, which I don't have time to do.

File: src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt [hunk 4]
```diff
@@ -28,8 +28,7 @@ object CustomNbtEvents: PaperBinFeature {
 	}
 	
 	override fun registerTransformers() {
-		return
-		register("net.minecraft.server.v1_12_R1.Entity") { classNode ->
+		return register("net.minecraft.server.v1_12_R1.Entity") { classNode ->
```
Comment (x4e) [inline src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt]: The previous formatting was intentional and has a different meaning to the new formatting.

**Review by x4e — CHANGES_REQUESTED** [review]
Thanks, some small changes needed

File: README.md (hunk 1 above)
Comment (dG9hc3Q) [reply README.md]: correct grammar

File: README.md (hunk 2 above)
Comment (dG9hc3Q) [reply README.md]: Alright

File: README.md (hunk 3 above)
Comment (dG9hc3Q) [reply README.md]: Alright

File: src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt (hunk 4 above)
Comment (dG9hc3Q) [reply src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt]: Alright, thought it was a typo

File: README.md (hunk 1 above)
Comment (x4e) [reply README.md]: The previous grammar was correct, new grammar is not

File: src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt (hunk 4 above)
Comment (x4e) [reply src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt]: The transformer is not being used so the `return` was to exit it before doing anything.
New format would still run the `register` body.

Comment (dG9hc3Q) [reply src/main/kotlin/dev/binclub/paperbin/transformers/CustomNbtEvents.kt]: 👍

**Review by x4e — APPROVED** [review]
